### DISCUSSION TITLE
Supabase auth redirect url issue

### DISCUSSION THREAD

**Discussion by hafiztblr** [supabase/supabase#13564] [Questions]
I am using supabase google auth in my project. when i use this auth locally it is redirected to http://localhost:3000/ after the completion of auth. But while i deploy this project to vercel and use the auth login from there it should expected to be redirected to that vercel url (where i deploy this project). but it is now redirected to localhost:3000 itself.

I have added my vercel url in Redirect URLs  authentication --> URL Configuration . But still it is not working

**Comment by GaryAustin1** [comment]
You need to adjust your redirectTo or emailRedirectTo option in the auth calls AND have the exact same URLs (full URL including http....) in the Redirect URL's dashboard.

**Reply by hafiztblr** [reply]
Ok..thanks for the reply @GaryAustin1
